IPython/utils/process.py

```diff
@@ -41,6 +41,9 @@ def find_cmd(cmd):
         from IPython.utils.process import pycmd2argv
         argv = pycmd2argv(get_ipython_module_path('IPython.terminal.ipapp'))
 
+    Note, The code for pycmd2argv has been removed now as it was not used
+    anywhere. get_ipython_module_path should give us that same result.
```

Comment:
No problem. No, you don't need to use `get_ipython_module_path` there. The `-m` flag takes the importable name of a module (like `IPython.terminal.ipapp`), not the path (like `.../IPython/terminal/ipapp.py`).

Using `sys.executable` means it will run with the same Python interpreter, which should find the same copy of IPython you're calling it from. There are ways to break that, but we needn't worry about them.